org.verapdf/src/org/verapdf/generator/ModelGenerator.xtend

```diff
@@ -52,6 +52,11 @@ class ModelGenerator implements IGenerator {
 			)
 		}
 		
+		fsa.generateFile(
+		"org/verapdf/model/GenericModelObject.java",

```

Comment:
Consider the use of constants for string paths such as the above. If there are a number of related constants then we should follow a convention for grouping them.